tests/golos.publication_tests.cpp

```diff
@@ -50,16 +50,18 @@ class golos_publication_tester : public golos_tester {
 
         produce_block();
         create_accounts(_users);
-        create_accounts({cfg::charge_name, cfg::token_name, cfg::vesting_name, cfg::emission_name, cfg::control_name, N(dan.larimer)});
+        create_accounts({cfg::charge_name, cfg::token_name, cfg::emission_name, cfg::control_name, N(dan.larimer)});
         produce_block();
 
         install_contract(_code, contracts::posting_wasm(), contracts::posting_abi());
-        install_contract(cfg::vesting_name, contracts::vesting_wasm(), contracts::vesting_abi());
         install_contract(cfg::charge_name, contracts::charge_wasm(), contracts::charge_abi());
         install_contract(cfg::token_name, contracts::token_wasm(), contracts::token_abi());
     }
 
     void init() {
+        create_account(cfg::vesting_name);
+        install_contract(cfg::vesting_name, contracts::vesting_wasm(), contracts::vesting_abi());
```

Comment:
Why did you have to bring these actions to this function? In my opinion, they should be executed in the constructor. Return them to the old place, if there is no reason to transfer them.